Branch-SDK/build.gradle

```diff
@@ -6,6 +6,7 @@ def jarFileName
 dependencies {
     compile fileTree(dir: 'libs', include: '*.jar')
     compile 'com.android.support:support-annotations:22.2.0'
+    compile(name:'answers-shim', ext:'jar')
```

Comment:
@stevewilber just noticed that there is change in package name than from the jar used earlier 
`com.crashlytics.sdk.android.answers.shim.AnswersOptionalLogger`  to `com.crashlytics.android.answers.shim.AnswersOptionalLogger`. Is that intentional?